pycbc/pnutils.py

```diff
@@ -797,3 +797,141 @@ def t2_cutoff_frequency(m1, m2, chi1, chi2):
 
 t4_cutoff_velocity = meco_velocity
 t4_cutoff_frequency = meco_frequency
+
+########################## This equation comes from Miriam (Mathematica) #######
+
+# Hybrid MECO in arXiv:1602.03134
+# To obtain the MECO, find minimum in v of eq. (6)
+
+def hybridEnergy(v, m1, m2, chi1, chi2, qm1, qm2):
+    """Return the hybrid energy [eq. (6)] whose minimum defines the hybrid MECO
+    up to 3.5PN (including the 3PN spin-spin)
+
+    Parameters
+    ----------
+    m1 : float
+        Mass of the primary object in solar masses.
+    m2 : float
+        Mass of the secondary object in solar masses.
+    chi1: float
+        Dimensionless spin of the primary object.
+    chi2: float
+        Dimensionless spin of the secondary object.
+    qm1: float
+        Quadrupole-monopole term of the primary object (1 for black holes).
+    qm2: float
+        Quadrupole-monopole term of the secondary object (1 for black holes).
+
+    Returns
+    -------
+    h_E: float
+        The hybrid energy as a function of v
+    """
+
+    pi = numpy.pi
+
```

Comment:
Define variables for v**2, v**4, chi1**2, etc. here, so you don't recompute them in each term; e.g., `v4 = v**4`, then use `v4` everywhere below.